third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2016,12 +2019,15 @@ func expandMasterAuth(configured interface{}) *containerBeta.MasterAuth {
 		Username: masterAuth["username"].(string),
 		Password: masterAuth["password"].(string),
 	}
+
 	if _, ok := masterAuth["client_certificate_config"]; ok {
```

Comment:
While you're here, let's use this as an opportunity to clean up the code a bit by putting masterAuth["client_certificate_config"] into a var